helm-charts/splunk-otel-collector/templates/config/_otel-collector.tpl

```diff
@@ -194,7 +197,13 @@ service:
         {{- if .Values.environment }}
         - resource/add_environment
         {{- end }}
-      exporters: [sapm]
+      exporters:
+        {{- if (eq (include "splunk-otel-collector.o11yTracesEnabled" .) "true") }}
+        - sapm
+        {{- end }}
+        {{- if (eq (include "splunk-otel-collector.platformTracesEnabled" .) "true") }}
+        - splunk_hec/platform_traces
+        {{- end }}
```

Comment:
Similar changes need to be done in the agent config. The gateway is not enabled by default

Reply:
good spot!  I updated the agent config (we switched to running in gateway mode hence not spotting it)